java/com/epam/ta/salad/Repository/SQLService/SaladServiceImpl.java

```diff
@@ -1,4 +1,51 @@
 package com.epam.ta.salad.Repository.SQLService;
 
-public class SaladServiceImpl {
-}
+import Entities.Salad;
+import Entities.Vegetable;
+import com.epam.ta.parking.util.NullCheckUtil;
+import com.epam.ta.salad.Repository.Repository.SaladRepository;
+import com.epam.ta.salad.Repository.RepositorySpecification.VegetableSpecification;
+import com.epam.ta.salad.Repository.SQLRepository.VegetableRepositoryImpl;
+import com.epam.ta.salad.Repository.Service.SaladService;
+import exceptions.NullVegetableException;
+
+import java.sql.SQLException;
+import java.util.List;
+
+public class SaladServiceImpl implements SaladService {
+    @Override
+    public Salad add(Salad salad) throws SQLException, ClassNotFoundException {
+        return null;
+    }
+
+    @Override
+    public Salad update(Salad salad) throws SQLException {
+        return null;
+    }
+
+    @Override
+    public Salad remove(Salad salad) throws SQLException {
+        return null;
+    }
+
+    @Override
+    public List<Vegetable> getObjects(Salad salad) throws SQLException, NullVegetableException {
```

Comment:
service mustn't to have method getObjects, it must to have many methods for each case which can be needed to user

Reply:
actually i don't know why you use salad as a method parameter

Reply:
Ok, if i will remove it from salad, and inplement read and get object?
![image](https://user-images.githubusercontent.com/46902048/69723241-2711d480-112a-11ea-9cac-130d30ad602f.png)
![image](https://user-images.githubusercontent.com/46902048/69723248-2b3df200-112a-11ea-8eff-819553f574ae.png)